src/resmom_win/mom_stage_file.c

```diff
@@ -295,16 +315,19 @@ main(int argc, char *argv[])
 		rmjobdir(rqcpf->rq_jobid, pbs_jobdir, NULL, NULL);
 	}
 
-	/* log the number of files/directories copied and the time it took */
-	copy_stop = copy_stop - copy_start;
-	sprintf(log_buffer, "staged %d items %s over %d:%02d:%02d",
-		num_copies, (dir == STAGE_DIR_OUT) ? "out" : "in",
-		copy_stop/3600, (copy_stop%3600)/60, copy_stop%60);
-	log_event(PBSEVENT_DEBUG2, PBS_EVENTCLASS_JOB, LOG_DEBUG,
-		rqcpf->rq_jobid, log_buffer);
+	/* if operation is successful, log the number of files/directories copied and the time it took */
+	if (!rc) {
+		copy_stop = copy_stop - copy_start;
+		sprintf(log_buffer, "staged %d items %s over %d:%02d:%02d",
+			num_copies, (dir == STAGE_DIR_OUT) ? "out" : "in",
+			copy_stop/3600, (copy_stop%3600)/60, copy_stop%60);
+		log_event(PBSEVENT_ERROR, PBS_EVENTCLASS_JOB, LOG_ERR, rqcpf->rq_jobid, log_buffer);
+	}
+
 	if ((stage_inout.bad_files) || (stage_inout.sandbox_private && stage_inout.stageout_failed)) {
 		if (stage_inout.bad_files) {
-			printf("%s\n", stage_inout.bad_list);
+			sprintf(log_buffer, "%s;stage %s failed for the file %s", id, (dir == STAGE_DIR_OUT) ? "out" : "in", stage_inout.bad_list);
```

Comment:
Nit: "stage%s" as opposed to "state %s".
That's how you have used it at other places. Upto you.